src/components/Books.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import InputBook from './InputBook';
+import ListBox from './ListBox';
+import Book from './Book';
+
+class Books extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      book: [],
```

Comment:
- [ ] [OPTIONAL] This is an array of books,  so the right name for this variable should be _books_.